jovian/jovian_nb_ext/main.js

```diff
@@ -438,123 +410,108 @@ define([
-            .addClass("form-check-input")
-            .attr("type", "radio")
-            .attr("name", "create_opt")
-            .attr("value", "False")
-        )
-        .append(
-          $("<label/>")
-            .addClass("form-check-label")
-            .text("False")
-        );
+      const create_new_label = $("<label/>").text("To create a new project?");
+      const create_new_box = $("<select/>")
+        .attr("id", "if_new")
+        .addClass("form-control")
+        .append($("<option/>").text("True"))
+        .append($("<option/>").text("False"))
+        .css("margin-left", "0em");
+
+      const secret_label = $("<label/>").text(
+        "Choose the type of notebook when creating a new project"
+      );
+      const secret_box = $("<select/>")
+        .attr("id", "nb_opt")
+        .addClass("form-control")
+        .append($("<option/>").text("auto"))
+        .append($("<option/>").text("public"))
+        .append($("<option/>").text("secret"))
+        .append($("<option/>").text("private"))
+        .css("margin-left", "0em");
+
       const artifacts_label = $("<label/>").text(
-        "Any outputs files or artifacts generated from the modeling processing. This can include model weights/checkpoints, generated CSVs, images etc. - Pass the list of strings(filenames) such as `submission.csv, weights.h5`"
+        "Any outputs files or artifacts generated from the modeling processing such as `submission.csv, weights.h5`"
       );
       const artifacts_box = $("<input/>")
         .addClass("form-control")
         .attr("id", "artifacts_box")
         .attr("placeholder", "submission.csv, weights.h5")
         .val("");
 
+      const git_commit_label = $("<label/>").text(
+        "To perform a Git commit? (only when the notebook is inside a Git repository)"
+      );
+      const git_commit_box = $("<select/>")
+        .attr("id", "if_git")
+        .addClass("form-control")
+        .append($("<option/>").text("True"))
+        .append($("<option/>").text("False"))
+        .css("margin-left", "0em");
+
+      const git_message_label = $("<label/>").text("Commit message for git");
+      const git_message_box = $("<input/>")
+        .addClass("form-control")
+        .attr("id", "git_msg_box")
+        .attr("placeholder", "Any message");
+
       div
-        .append(secret_label)
-        .append(secret_box)
+        .append(message_label)
+        .append(message_box)
         .append("<br>")
         .append(nb_filename_label)
         .append(nb_filename_box)
         .append("<br>")
         .append(files_label)
         .append(files_box)
         .append("<br>")
-        .append(capture_env_label)
-        .append(capture_env_box)
-        .append("<br>")
         .append(env_type_label)
         .append(env_type_box)
         .append("<br>")
-        .append(notebook_id_label)
-        .append(notebook_id_box)
+        .append(project_id_label)
+        .append(project_id_box)
```

Comment:
Move this after `create_new` as project_id is dependent on create_new (disables when create_new is True)